apps/system/lockscreen/js/state/lockscreen_basic_state.js

```diff
@@ -57,6 +61,17 @@
    * Or the component cannot track the last active state.
    */
   LockScreenBasicState.prototype.transferTo = function() {
+    if (this._transferred) {
```

Comment:
I am not convinced we need this flag. It sounds like a workaround to a deeper issue.